### ISSUE TITLE
Tab stuck on the Please wait

### ISSUE THREAD

**Issue by ChrisK40** [OfficeDev/microsoft-teams-emergency-operations-center#63]
Hi Guys, I been following deployment to the dot.  when i try to open https://xxxx.azurewebsites.net/#/tab  its  stuck on please wait scree. 
 
I have found the following in F12 console , but i can confirm that my browser has javascript enabled. 

![image](https://user-images.githubusercontent.com/120035744/206212950-f0e74faf-33d8-4462-ab6e-edd1f6a6fd0c.png)

**Comment by v-prigunasek** [comment]
Hi @ChrisK40 ,

Thanks for submitting an issue here.
Can you please confirm whether the deployment is successful in Azure Portal? If so, could you please open the TEOC app in browser and share the console log file to assist you further on this issue?

**Comment by ChrisK40** [comment]
Hi @v-prigunasek 

Please find screenshot below. 

![image](https://user-images.githubusercontent.com/120035744/206390183-2f3b614d-b27e-4294-96e9-576eb1ce6811.png)

**Comment by v-prigunasek** [comment]
Hi @ChrisK40 ,

Thanks for sharing this.  Could you please open the TEOC app in browser and share the console log file?

**Comment by ChrisK40** [comment]
Hi @v-prigunasek 

when i open app in the browser i get Please wait ....  Could you  tell me were i can get those logs from ?

**Comment by v-prigunasek** [comment]
Yes, that is correct. Are you seeing any other log messages or errors in the console? This image does not have any information.

**Comment by ChrisK40** [comment]
@v-prigunasek  Thats what i got from edge console 

![image](https://user-images.githubusercontent.com/120035744/206392028-062bb697-993d-48e2-98d7-8891aa36eec5.png)

**Comment by v-prigunasek** [comment]
Hi @ChrisK40 ,

By any chance, have you tried to launch the app in Teams Desktop application to see if it works? Also, can you please clear the cache in browser and try to launch the app again?

**Comment by ChrisK40** [comment]
Hi @v-prigunasek , 

still no luck ...

**Comment by v-prigunasek** [comment]
Hi @ChrisK40 ,

Can you please share the HAR file from the browser? 
1. Open the TEOC app in the browser 
2. Open the Developer Tools 
3. Select the Network Tab
4. Select the Export HAR... in the toolbar to export the trace as a "HAR" file

**Comment by ChrisK40** [comment]
Done,  I cant upload a HAR type so i will just zip it. 
[azurewebsites.net.zip](https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/files/10185199/azurewebsites.net.zip)

**Comment by v-prigunasek** [comment]
Hi @ChrisK40 ,

Can we get into a quick call to troubleshoot this issue? Please let me know your availability and email contact to send you an invite?

**Comment by ChrisK40** [comment]
Hi @v-prigunasek , 

Chris.krycia@cloudpay.net 

I'm free for the next 4 hrs if that works for you . (Teams ?) 

Regards
Chris
